SoObjects/SOGo/SOGoGCSFolder.m

```diff
@@ -927,30 +927,34 @@ - (BOOL) subscribeUserOrGroup: (NSString *) theIdentifier
   NSMutableArray *allUsers;
   SOGoUserSettings *us;
   NSDictionary *dict;
-  SOGoUser *sogoUser;
   BOOL rc;
   int i;
 
   dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: theIdentifier];
```

Comment:
I think you should limit the lookup to the active user's domain here because you don't do it later.

Reply:
Resolved in commit 948c89f